### PR TITLE
done with first 13 files

### PR DESCRIPTION
**Pull request by tlaudahl**

### CODE REVIEW

**Comment by mjherich** [comment]
Good job so far @tlaudahl ! Once you hit MVP add a comment here and @ me.

**Comment by tlaudahl** [comment]
@mjherich I got MVP done

File: src/15_classes.py [hunk 1]
```diff
@@ -2,19 +2,41 @@
...
+    def __str__(self):
+        return f'Lat: {self.lat}, Long: {self.lon}, Name: {self.name}'
 
 # YOUR CODE HERE
 
 # Make a class Geocache that can be passed parameters `name`, `difficulty`,
 # `size`, `lat`, and `lon` to the constructor. What should it inherit from?
-
+class Geocache(Waypoint):
+    # def __init__(self, name, difficulty, size, lat, lon):
```
Comment (mjherich) [inline src/15_classes.py:26]: Glad Brady went over this during lecture so you know why it wasn't working!

**Review by mjherich — APPROVED** [review]
Good job on this project, everything looks good 👍

**Pull request merged**